.taskcluster.yml

```diff
@@ -494,3 +494,49 @@ tasks:
           type: file
       cache:
         generic-worker-checkout: /go/src
+
+  - provisionerId: "{{ taskcluster.docker.provisionerId }}"
+    workerType: "{{ taskcluster.docker.workerType }}"
+    metadata:
+      name: dep-check
+      description: "Run 'dep check' to ensure dependencies are up to date"
+      owner: "{{ event.head.user.email }}" # the user who sent the pr/push e-mail will be inserted here
+      source: "{{ event.head.repo.url }}"  # the repo where the pr came from will be inserted here
+    extra:
+      github:
+        # Events that will trigger this task
+        events:
+          - pull_request.opened
+          - pull_request.synchronize
+          - push
+    scopes:
+      - secrets:get:repo:github.com/taskcluster/generic-worker
+      - docker-worker:cache:generic-worker-checkout
+    payload:
+      features:
+        taskclusterProxy: true
+      maxRunTime: 3600
+      image: golang:1.10
+      env:
+        # once bug 1508383 is landed, this won't be required (the worker will supply it)
+        TASKCLUSTER_ROOT_URL: https://taskcluster.net
```

Comment:
I think this isn't needed as bug 1508383 has landed, but I'll remove it in a separate PR as we have that throughout the `.taskcluster.yml` file.

Reply:
It is still needed for generic-worker workers that have not been upgraded to ^13.0.0